.github/workflows/tekton_bundle_push.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: Tekton Bundle Push
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['main']
+  workflow_dispatch:
+env:
+  BUNDLE: hello-world/hello-world.yaml
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: hacbs-release/hello-world
+  IMAGE_TAG: 0.3-alpine
+  REGISTRY_USER: ${{ secrets.QUAY_ROBOT_USER }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY_ROBOT_TOKEN }}
+jobs:
+  run-pipeline:
+    name: Tekton Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: debianmaster/actions-k3s@master
```

Comment:
What for?

Reply:
Here is a reproducer:
```
$ tkn bundle list -o yaml quay.io/hacbs-release/hello-world:0.3-alpine
Error: Couldn't get kubeConfiguration namespace: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
```

Apparently Tekton CLI does not function without kubernetes kubectl clients installed, and configured. I played around with the `KUBERNETES_MASTER` environment variable, but couldn't figure out how to make it work.

I was surprised by this, and welcome any suggestion to workaround or eliminate the dependency. I very much want to drop this part of the workflow!

Reply:
Thx @scoheb for helping solve this issue.
(we moved to using fake kube configs)